app/Http/Requests/BulkUpdateRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Enums\RoomUserRole;
+use App\Models\User;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class BulkUpdateRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'role'    => ['required',Rule::in([RoomUserRole::USER,RoomUserRole::MODERATOR,RoomUserRole::CO_OWNER])],
+            'users'   => ['required','array'],
+            'users.*' => ['required','integer','exists:App\Models\User,id',
```

Comment:
Add distinct to ensure no duplicate ids
```suggestion
            'users.*' => ['required','integer','distinct','exists:App\Models\User,id',
```